src/components/Reports/GoalCalculator/SummaryReport/MpdGoal/MpdGoalHeaderCards/MpdGoalHeaderCards.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Box, Card, Typography, styled } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+import { useLocale } from 'src/hooks/useLocale';
+import { currencyFormat, percentageFormat } from 'src/lib/intlFormat';
+import { Goal } from '../MpdGoalTable';
+import { useGoalLineItems } from '../useGoalLineItems';
+
+const StyledCard = styled(Card)(({ theme }) => ({
+  flex: 1,
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  padding: theme.spacing(4),
+  overflowX: 'auto',
+  whiteSpace: 'nowrap',
+}));
+
+const StyledTypography = styled(Typography)(({ theme }) => ({
+  fontWeight: 'bold',
+  color: theme.palette.primary.main,
+  fontSize: theme.typography.h2.fontSize,
+  [theme.breakpoints.down('sm')]: {
+    fontSize: theme.typography.h4.fontSize,
+  },
+}));
+
+interface MpdGoalHeaderCardsProps {
+  goal: Goal;
+}
+
+export const MpdGoalHeaderCards: React.FC<MpdGoalHeaderCardsProps> = ({
+  goal,
+}) => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const locale = useLocale();
+  const { overallTotal, supportRaisedPercentage } = useGoalLineItems(goal);
+
+  return (
+    <Box
+      height={theme.spacing(25)}
+      display="flex"
+      flexDirection="row"
+      justifyContent="space-between"
+      gap={theme.spacing(4)}
+    >
```

Comment:
If this `Box` is mostly just to put the cards next to each other, you may be able to use a `<Stack direction="row">` and remove the flex styles.